src/templates/static/get_started/index.jsx

```diff
@@ -42,6 +43,70 @@ const CTA = () => (
     </div>
 );
 
+const VolatilityIndicesTable = () => {
+    const active_symbols = [
+        {
+            key             : 'r10',
+            label           : it.L('Volatility Indices 10'),
+            high_close_value: 86.85,
+            close_low_value : 87.91,
+            high_low_value  : 87.38,
+        },
+        {
+            key             : 'r_25',
+            label           : it.L('Volatility Indices 25'),
+            high_close_value: 88.24,
+            close_low_value : 86.92,
+            high_low_value  : 87.59,
+        },
+        {
+            key             : 'r50',
+            label           : it.L('Volatility Indices 50'),
+            high_close_value: 87.21,
+            close_low_value : 87.87,
+            high_low_value  : 87.54,
+        },
+        {
+            key             : 'r75',
+            label           : it.L('Volatility Indices 75'),
+            high_close_value: 87.88,
+            close_low_value : 87.27,
+            high_low_value  : 87.58,
+        },
+        {
+            key             : 'r100',
+            label           : it.L('Volatility Indices 100'),
+            high_close_value: 87.64,
+            close_low_value : 87.35,
+            high_low_value  : 87.49,
+        },
+    ];
+    return (
+        <div className='gr-padding-30 gr-parent'>
+            <table>
+                <thead>
+                    <tr>
+                        <th className='align-start'>{it.L('Volatility Indices')}</th>
+                        <th className='align-end'>{it.L('High - Close')}</th>
+                        <th className='align-end'>{it.L('Close - Low')}</th>
+                        <th className='align-end'>{it.L('High - Low')}</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {active_symbols.map(item => (
+                        <tr key={item.key}>
+                            <td>{item.label}</td>
+                            <td className='align-end'>{Math.round(item.high_close_value)}</td>
```

Comment:
why not centred? it looks a bit weird with align end

Reply:
The text label is way wider than the values, From the excel days, we always formatted the numbers that matter to the right. I can do center too but I preferred this one :smile:. 
If you think we should go with the center, say so.